release-notes/UWPSupportedFeatures-v6.1.md

```diff
@@ -0,0 +1,60 @@
+WCF Features in UWP 6.1
+======================================================
+*UWP 6.1 features are based on .NET Core 2.0. This is part of [WCF release notes](https://github.com/dotnet/wcf/releases/tag/v2.0.0) for .NET Core 2.0. Please see the release notes for what's new since 1.1.0.*
```

Comment:
Technically, UWP 6.1 is not part of .NET Core 2.0 release. Maybe it's worth mentioning UWP 6.1 is part of VS 2017 v15.7 (pls double check the version). Maybe you can find an official link of UWP 6.1 release.